packages/builder/src/components/automation/AutomationBuilder/FlowChart/FlowItemHeader.svelte

```diff
@@ -69,12 +111,32 @@
       <div class="iconAlign">
         {#if isTrigger}
           <Body size="XS"><b>Trigger</b></Body>
-          <Body size="XS">When this happens:</Body>
         {:else}
           <Body size="XS"><b>Step {idx}</b></Body>
-          <Body size="XS">Do this:</Body>
         {/if}
-        <Detail size="S">{block?.name?.toUpperCase() || ""}</Detail>
+        <input
+          placeholder="Enter some text"
+          name="name"
+          autocomplete="off"
+          value={automationName}
+          on:input={e => {
+            automationNameError = getAutomationNameError(e.target.value)
```

Comment:
We also might want to trim the value here, because in `saveName` we depend on checking the length of the name, and we wouldn't want to store a blank name that's just a space for example.